application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/RevocableRoleBasedAction.java

```diff
@@ -0,0 +1,50 @@
+package org.togetherjava.tjbot.commands.moderation.temp;
+
+import net.dv8tion.jda.api.requests.ErrorResponse;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link RevocableModerationAction} abstract implementation for a role based moderation action.
```

Comment:
That javadoc is obsolete and doesnt add anything to the reader, not helpful. It is basically identical to the first line of code `abstract class RevocableRoleBasedAction implements RevocableModerationAction`.

Instead, you should take the opportunity and explain the use case, what this is.

> Role based moderation actions that can be revoked, for example a {@link ...Mute...} or {@link ...Quarantine...} which are applied implicitly purely by the presence of a role.

Reply:
changed it.